### ISSUE TITLE
Sort album tracks by track number

### ISSUE THREAD

**Issue by xbow** [lfcabend/mopidy-spotify-web#4]
Currently, the tracks of an album do not seem to be ordered by track number. Could this be implemented?

**Comment by alligatortower** [comment]
I would also love it if this could be added.

**Comment by emilithe** [comment]
In `ncmpcpp` client it is possible to define sorting rules via parameters (for more details see `man ncmpcpp`)

```
browser_sort_mode
browser_sort_format
```

I tested what `ncmpc` client does and it seems it shows track list in order as received from mopidy.

I've checked mopidy response to client while browsing albums.
For single-disk albums it works good - mopidy-mpd frontend sends list already sorted by track number: 
[singledisk_debug-log.txt](https://github.com/lfcabend/mopidy-spotify-web/files/72860/singledisk_debug-log.txt)

But for multiple-disks albums sorting partially fails:
[multidisk_debug-log.txt](https://github.com/lfcabend/mopidy-spotify-web/files/72859/multidisk_debug-log.txt)

I wonder if it is possible to grab disk number from spotify, so mpd mopidy frontend and mpd clients may property sort the track list for multidisk albums.
